test/e2e/framework.go

```diff
@@ -636,19 +703,19 @@ func deleteYAML(filename string, ns string) error {
 	return nil
 }
 
-func (data *TestData) waitForResourceExistOrNot(namespace string, resourceType string, resourceName string, shouldExist bool) error {
+func (data *TestData) waitForResourceExist(namespace string, resourceType string, key string, value string, shouldExist bool) error {
```

Comment:
I believe this func also used in other e2e test, need know other component's owner to modify accordingly.

Reply:
the original methods still exist, it is wrapped to extend for more parameters. see line 737